qcore/tests/test_helpers.py

```diff
@@ -250,3 +251,17 @@ def __repr__(self):
     assert_eq('\'a\'', qcore.safe_repr('a'))
     assert_eq('\'...', qcore.safe_repr('2.192842', max_length=4))
     assert_eq('<n/a: repr...', qcore.safe_repr(TestObject(), max_length=13))
+
+
+def test_marker_object():
+    assert_eq('text', six.text_type(qcore.MarkerObject('text')))
```

Comment:
unicode string on left? do we also want to ensure type is same for these equal checks?